tests/product/test_server_install.py

```diff
@@ -193,7 +202,7 @@ def test_install_with_java8_home(self):
         expected = installed_all_hosts_output
 
         actual = cmd_output.splitlines()
-        self.assertEqual(sorted(expected), sorted(actual))
+        self.assertRegexpMatchesLineByLine(actual, expected)
```

Comment:
Why these changes? I've long thought that this was to deal with the fact that there are no guarantees about what order of the actual output will be since there are 3-4 processes running in parallel.